components/atom/videoPlayer/demo/index.js

```diff
@@ -0,0 +1,5 @@
+import AtomVideoPlayer from 'components/atom/videoPlayer/src'
+// export default () => <AtomVideoPlayer src='https://www.youtube.com/watch?v=1gI_HGDgG7c' />
```

Comment:
why is this line comented?

Reply:
Try to follow the same react template pattern as others demos does (take a look on the padding and title, description, ...)